dao/video-section.go

```diff
@@ -0,0 +1,34 @@
+package dao
+
+import (
+	"github.com/joschahenningsen/TUM-Live/model"
+)
+
+func CreateVideoSection(section *model.VideoSection) error {
+	return DB.Create(&section).Error
+}
+
+func CreateVideoSectionBatch(sections []model.VideoSection) error {
+	var err error
+	for _, section := range sections {
+		err = CreateVideoSection(&section)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
```

Comment:
Why not do it like this?

```suggestion
func CreateVideoSections(sections []model.VideoSection) error {
	return DB.Create(&sections).Error
}
```

Reply:
That's a pretty good question. Guess my brain wasn't working properly again. 😅 Thanks for pointing out!